.gitignore

```diff
@@ -1,5 +1,4 @@
 .DS_Store
-bower_components
```

Comment:
Should we really delete `bower_components` from `.gitignore` ?
We do not create the folder, but if one exists, it should still be ignored.